Maki.gg/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "Quintenvw",
+    "id": "123468845749895170"
+  },
+  "service": "Maki",
```

Comment:
Server name should be the same as folder name

Reply:
https://uwu.sk/u/Ca.png
https://uwu.sk/u/LJ.png